src/Tizen.NUI/src/public/Visuals/VisualFactory.cs

```diff
@@ -136,5 +136,44 @@ public void UsePreCompiledShader()
             Interop.VisualFactory.UsePreCompiledShader(SwigCPtr);
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
+
+        /// <summary>
+        /// Adds a list of pre-compiled shaders to the visual factory.
+        /// </summary>
+        ///
+        /// This API allows you to add the desired precompile shader to the list.
+        /// you can set it through PropertyMap.
+        /// you need to know the values for setting well to use them, so please refer to the explanation below.
+        ///
+        /// The property map consists of string keys.
+        ///
+        /// - shaderType: Set the desired shader type. we provides these type: "image","text","color","3d" and "custom"
+        /// - shaderOption(propertyMap): Set the desired shader option. we provides these flag: we provides a lot of shader options, so user need to check proper shader option.
+        /// - vertexShader: Set the vertext shader that user want. this is for custom shader.
+        /// - fragmentShader: Set the fragment shader that user want. this is for custom shader.
+        /// - shaderName: if user want to set shader name, use this. this is for custom shader.(optional)
+        ///
+        /// (example)
+        /// PropertyMap imageShader = new PropertyMap();
+        /// imageShader.Add("shaderType", new PropertyValue("image"));
+        /// imageShader.Add("shaderOption", new PropertyValue(new PropertyMap().Add("ROUNDED_CORNER", new PropertyValue(true))
+        ///                                                                     .Add("MASKING", new PropertyValue(true))));
+        ///
+        /// PropertyMap textShader = new PropertyMap();
+        /// textShader.Add("shaderType", new PropertyValue("text"));
+        ///
+        /// VisualFactory.Instance.AddPrecompileShader(imageShader);
+        /// VisualFactory.Instance.AddPrecompileShader(textShader);
+        /// VisualFactory.Instance.UsePreCompiledShader();
+        ///
+        /// <param name="option">The map contains the shader option for precompiling.</param>
+        [EditorBrowsable(EditorBrowsableState.Never)]
```

Comment:
If this method really needs the boolean return, could you leave a description about the return meaning as well?
Like,
```suggestion
        /// <return>True if the pre-compiled shader is added, otherwise false.</return>
        [EditorBrowsable(EditorBrowsableState.Never)]
```

Reply:
감사합니다.
리뷰 남겨주신대로 수정 완료했습니다 👍

Reply:
빠른 반영 감사합니다~